communities.json

```diff
@@ -743,6 +743,13 @@
       "inviteCode": "tokio",
       "githubUrl": "https://github.com/tokio-rs"
     },
+    {
+      "logo": "multitheftauto.png",
```

Comment:
After merging, your project will be featured on our [Discord ❤ Open Source](https://discord.com/open-source) page. It automatically applies a filter to your logo that makes it solid white except for the transparent portions.

The image you've included will look incorrect when this filter is applied. Could you swap the image out for a version with a transparent background that will look correct when solid white on a dark background?